### ISSUE TITLE
How to change bluetooth device

### ISSUE THREAD

**Issue by AdamWeglarz** [breitburg/python-am43#5]
Hi,

I would like to use external bluetooth device.
I cannot see anywhere in code how to choose HCI device.
Devices:
        hci1    00:19:0E:11:AC:CC
        hci0    B8:27:EB:1A:A6:C5

Can you help me?

Adam

**Comment by breitburg** [comment]
Hello! Okay, I'll try to do something with it today 👌

**Comment by AdamWeglarz** [comment]
Great :-) waiting for it :-)

**Comment by breitburg** [comment]
Sorry for a delay 😬

According to a [`Scanner` class documentation](https://ianharvey.github.io/bluepy-doc/scanner.html) you can specify the index of your HCI device (it's zero by default). For now you can edit [5th line in the `search.py` file](https://github.com/breitburg/python-am43/blob/62708960f43bc20460847c907a20ab8e4cdd518e/am43/search.py#L5) and specify it manually.

Later this week I will release a new version with that feature implemented as an optional parameter for `am43.search` method. Thank you for using my library 🙃

**Comment by breitburg** [comment]
I had released a new version 1.1.0 with that feature included.

Now you can specify your device like that:
```python
import am43
blind = am43.search('xx:xx:xx:xx:xx:xx', device=1)
```
